Formula/libcouchbase@2.rb

```diff
@@ -19,6 +19,8 @@ class LibcouchbaseAT2 < Formula
 
   keg_only :versioned_formula
 
+  deprecate! date: "2023-01-20", because: :versioned_formula
```

Comment:
Maybe `:deprecated_upstream` but not too sure so just went with `:versioned_formula`. The enterprise page mentions Couchbase C SDK 2.10 ended full maintenance mode on March 2021 and reached EOL on March 2022 https://www.couchbase.com/support-policy/enterprise-software/

Current v2.10.9 was released on March 3, 2021

Reply:
With that information it seems like this
```suggestion
  deprecate! date: "2023-01-20", because: : deprecated_upstream
```